### PR TITLE
Update build and test workflow

### PR DESCRIPTION
**Pull request by zmichaels11**
### Changes
* Update `setup-ros` to 0.0.13
* Update `action-ros-ci` to 0.0.13

### Notes
* Actions are currently failing due to Azure sync.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -13,11 +13,11 @@ jobs:
...
-    - uses: ros-tooling/action-ros2-ci@0.0.11
+    - uses: ros-tooling/setup-ros@0.0.13
+    - uses: ros-tooling/action-ros-ci@0.0.13
       with:
         package-name: ros2bag rosbag2_compression rosbag2_cpp rosbag2_storage rosbag2_storage_default_plugins rosbag2_test_common rosbag2_tests shared_queues_vendor sqlite3_vendor
     - uses: actions/upload-artifact@master
       with:
         name: colcon-logs
-        path: ros2_ws/log
+        path: ros_ws/log
```
Comment (Karsten1987) [inline .github/workflows/test.yml]: why this change?

Comment (zmichaels11) [reply .github/workflows/test.yml]: The GH Action `action-ros-ci` now builds in `ros_ws` instead of `ros2_ws`.

**Comment by zmichaels11** [comment]
Rebasing this onto master. I plan to merge this in if GH Actions are all green.
